.github/workflows/bi-connectors.yml

```diff
@@ -9,7 +11,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
Why did we need to switch it off of `ubuntu-latest`?

Reply:
To avoid any possible troubles when `ubuntu-latest` would be upgraded. See discussion thread there: https://github.com/Bit-Quill/opensearch-project-sql/pull/118#discussion_r971385524.